ref/android-features.md

```diff
@@ -0,0 +1,21 @@
+Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
+
+| Feature           | Supported | Notes  |
+|--------------------|:-----------------:|
+| [OpenGL ES](https://source.android.com/devices/graphics/arch-egl-opengl) <= 3.2   |  ✓  | |
+| [Vulkan](https://source.android.com/devices/graphics/arch-vulkan) |  ✓  | |
```

Comment:
Maybe we need to put something in the `Notes` column like `this feature is only available for Anbox Cloud deployment where AMD or Intel GPU is installed`

Reply:
I was thinking about that first but it makes things rather complex. We should keep this more generic and put details to https://anbox-cloud.io/docs/exp/gpu-support

Reply:
I'm fine with that.